src/core_ocean/shared/mpas_ocn_gm.F

```diff
@@ -275,10 +275,9 @@ subroutine ocn_GM_compute_Bolus_velocity(statePool, diagnosticsPool, &
             maxLocation = k
             do k = maxLocation, maxLevelCell(iCell)
                BruntVaisalaFreqTopEdge = max(BruntVaisalaFreqTop(k, iCell), 0.0_RKIND)
-
-               RediKappaScaling(k, iCell) = min(max(config_GM_min_stratification_ratio, &
-                                               BruntVaisalaFreqTopEdge/(maxN + 1.0E-10_RKIND)), &
-                                           1.0_RKIND)
+               RediKappaScaling(k, iCell) = min(max(config_Redi_N2_based_taper_min,                  &
```

Comment:
@vanroekel I was expecting to replace ratio with a min/max. Is the max assumed to be one here, so we just have the min? If so, then this is correct.

Reply:
yes, the max is just 1 and shouldn't be changed